README.md

```diff
@@ -82,28 +74,69 @@ ansible-galaxy collection build --force
 ansible-galaxy collection install cisco-aci-* --force
 ```
 
-### Second Approach
+## Use Cases
+
+Once the collection is installed, you can use it in a playbook by specifying the full namespace path to the module, plugin and/or role.
+
+### Adding a new EPG
 
-Go to [ansible-aci Actions](https://github.com/CiscoDevNet/ansible-aci/actions)
+```yml
+- hosts: aci
+  gather_facts: no
 
-Select the latest CI build
+  tasks:
+  - name: Add a new EPG
+    cisco.aci.aci_epg:
+      hostname: apic
+      username: admin
+      password: SomeSecretPassword
+      tenant: production
+      ap: intranet
+      epg: web_epg
+      description: Web Intranet EPG
+      bd: prod_bd
+    delegate_to: localhost
+```
 
-Under Artifacts download collection and unzip it using Terminal or Console.
+## Optimizing Playbooks
 
-*Note: The collection file is a zip file containing a tar.gz file. We recommend using CLI because some GUI-based unarchiver might unarchive both nested archives in one go.*
+There are two main methods to optimize the execution of ACI modules in your playbooks.
 
-Install the unarchived tar.gz file
+1. Using the ACI HTTPAPI plugin
+1. Using the `suppress_` options
 
-```sh
-ansible-galaxy collection install cisco-aci-1.0.0.tar.gz —-force
-```
+To find out more about optimizing playbook execution, please refer to the [Optimizing Playbooks](docs/optimizing.md) documentation.
 
-### See Also
+## Testing
 
-- [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
+Integration tests for each module in the cisco.aci collections are executed against the following ACI versions:
 
-## Contributing to this collection
+- 4.2(7s)
+- 5.2(7g)
+- 6.0(2h)
+
+## Contributing
 
 Ongoing development efforts and contributions to this collection are tracked as issues in this repository.
 
 We welcome community contributions to this collection. If you find problems, need an enhancement or need a new module, please open an issue or create a PR against the [Cisco ACI collection repository](https://github.com/CiscoDevNet/ansible-aci/issues).
+
+## Support
+
+This collection supports any ACI version within the Last Day of Support (LDOS) date.
+
+Some modules and options within the collection are only available from specific versions of ACI. The versions a module or option supports are documented within the individual module documentation.
+
+To find EOL announcements for ACI versions, refer to the [End-of-Life and End-of-Sale Notices](https://www.cisco.com/c/en/us/products/cloud-systems-management/application-policy-infrastructure-controller-apic/eos-eol-notice-listing.html) page.
+
+## Release Notes
+
+See the [Changelog](CHANGELOG.rst) for full release notes.
+
+## Related Information
+
+For additional information and guides see the [Cisco ACI DevNet documentation](https://developer.cisco.com/docs/aci/ansible/#cisco-aci-ansible-modules).
```

Comment:
Lets also add a link to : 
* https://developer.cisco.com/docs/nexus-as-code/aci-with-ansible/
* https://developer.cisco.com/learning/tracks/aci-programmability/ansible-aci-intro/